website/src/pages/case-studies/workiva-batch-changes.tsx

```diff
@@ -0,0 +1,115 @@
+                        <ul>
+                            <li>Propagate a new version of React to all frontend repos</li>
+                            <li>Update API versions of kubernetes resources</li>
+                            <li>Migrate to a new CDN while updating all code references</li>
+                            <li>Update UI component syntax as necessary to support a new version of the Dart language</li>
+                            <li>Run test batch changes to ensure that new chrome web browser releases wouldn't result in any software failures</li>
+                        </ul>
+                    </p>
+
+                    <InContentBlockquote
+                        quote="The fact that Batch Changes runs each step via Docker is super powerful. As long as there's a tool out there that does what you need, you can incorporate it into Batch Changes with minimal effort, and that's been awesome for us."
+                        author="Evan Weible, Staff Software Engineer, Workiva"
+                    />
+
+                    <h2 className="pt-5 pb-1">
+                        Workiva reduces the time it takes to make large-scale code changes by 80%
+                    </h2>
+
+                    <p>The Client Platform team has already used Batch Changes numerous times to propagate large-scale updates to the frameworks and shared libraries they maintain. In comparison to manually making these changes, Batch Changes reduces the time it takes to make large-scale code changes by 80%.</p>
+
+                    <InContentBlockquote
+                        quote="Updating all of our repositories with Batch Changes saves time, is less error-prone, and gives us confidence that everything is going to plan."
+                        author="Joe Bingham, Software Engineer, Workiva"
+                    />
+
+                    <p>Instead of spending time maintaining their internal tool, the Workiva team will be using Batch Changes going forward.</p>
+
+                    <InContentBlockquote
+                        quote="Using a tool that isn’t maintained by us saves us time and drives better adoption. In comparison to the tool we built internally, it's much easier to work with, more feature complete, and well documented."
+                        author="Evan Weible, Staff Software Engineer, Workiva"
+                    />
+                    <br/>
+                </div>
+            </ContentSection>
+        </CaseStudyPage>
+        <CaseStudyRequestDemoForm
+            title="Need help with your monolith to microservices migration?"
+            description="Sourcegraph Universal Code Search can ensure a safe and efficient deployment plan for migrating your monolith to microservices. Get in touch to schedule a demo."
```

Comment:
```suggestion
            description="Learn how companies of all sizes and in all industries use Sourcegraph to solve big code problems."
```

Reply:
@jeanduplessis - is it possible to link to the Batch Changes specific demo request form just for this case study?